src/webpack-plugin.ts

```diff
@@ -31,10 +48,33 @@ class VueAutoRoutingPlugin {
 
       fs.writeFileSync(to, code)
     }
+    const generateIndex = (options: Options[]) => {
+      let importCode = '', exportCode = '';
+      options.forEach(option => {
+        importCode += `import ${option.pageName} from './${option.pageName}';\n`
+        exportCode += `${option.pageName},`
+      })
+      return `${importCode}export{${exportCode}}`
+    }
 
     compiler.hooks.thisCompilation.tap(pluginName, compilation => {
       try {
-        generate()
+        if (Object.prototype.toString.call(this.options) === '[object Object]') {
+          generate(this.options as Options);
+        } else if (Array.isArray(this.options)) {
+          this.options.forEach(option => {
```

Comment:
We have to validate `pageName` here as we need it if multiple options are specified.

Reply:
fixed
```
} else if (Array.isArray(this.options)) {
         const isAvailable = this.options.every((value) => {
            return typeof value.pageName ==='string'
          })
          if(!isAvailable){
            throw new Error('MPA router option need pageName')
          }
```